tslint.json

```diff
@@ -119,7 +119,7 @@
     "no-output-on-prefix": true,
     "use-input-property-decorator": true,
     "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "use-host-property-decorator": false,
```

Comment:
I think this should not be turned off.

Reply:
Oh, sorry I was turning it to true, didn't realise it was already by taking reference from earlier branch.